ccd-definition/CaseEvent.json

```diff
@@ -195,8 +195,9 @@
     "Description": "Review and submit application",
     "DisplayOrder": 17,
     "PreConditionState(s)": "*",
```

Comment:
Pre condition should be set to `1_Initiation` so only in such state submit is possible.